moveit_ros/moveit_servo/config/servo_parameters.yaml

```
@@ -1,5 +1,28 @@
 servo:
-################################ ROBOT SPECIFIC CONFIG #############################
+################################ GENERAL CONFIG ################################
+
+  thread_priority: {
+    type: int,
+    read_only: true,
+    default_value: 40,
+    description: "This value is used when configuring the servo loop thread to use SCHED_FIFO scheduling \
+                  We use a slightly lower priority than the ros2_control default in order to reduce jitter \
+                  Reference: https://man7.org/linux/man-pages/man2/sched_setparam.2.html",
+    validation: {
+      gt_eq<>: 0
+    }
+  }
+
+  publish_period: {
+    type: double,
+    read_only: true,
+    default_value: 0.034,
+    description: " 1 / (Nominal publish rate) [seconds]",
+    validation: {
+      gt<>: 0.0
+    }
+  }
+
   move_group_name: {
     type: string,
     description: "The name of the moveit move_group of your robot \
```

Not possible to change at this time

```suggestion
    type: string,
    read_only: true,
    description: "The name of the moveit move_group of your robot \
```